dice.py

```diff
@@ -0,0 +1,23 @@
+import random
+
+repeat = True
+while repeat:
+    no = random.randint(1, 6)
+    print("You rolled", no)
+    if no == 1:
+        print("You feel somewhat disappointed")
+    if no == 2:
+        print("You look at the two dots and begin to mourn a lover lost at sea")
+    if no == 3:
+        print("Three is the magic number, you turn into a frog")
+    if no == 4:
+        print("4! You win...not really, you're just rolling a dice. Get some mates!")
+    if no == 5:
+        print("You bleed a little")
+    if no == 6:
+        print("You begin to wonder what this point of this is")
+    if no == 7:
+        print("err...what? This is literally impossible")
```

Comment:
Rather than use multiple if statements you would be better using
```
 if no == 1:
        print("You feel somewhat disappointed")
    elif no == 2:
    ...
    else no == 7:
        print("err...what? This is literally impossible")
```
Your script will currently keep checking all the conditions, even if it's already matched one. elif/else will skip the rest as soon as it finds a match so is faster